src/utils/accounts.ts

```diff
@@ -24,17 +24,21 @@ function getNpmConfigFileContent(token: string) {
 export async function addAuthTokenToNpmConfig(token: string) {
     const homeDirectory = os.homedir();
     const npmConfigFile = ".npmrc"
-    const npmConfigContent = getNpmConfigFileContent(token);
+    let npmConfigContent = getNpmConfigFileContent(token);
 
     if (fs.existsSync(path.join(homeDirectory, npmConfigFile))) {
         const npmConfigFileContent = await readUTF8File(path.join(homeDirectory, npmConfigFile));
         if (npmConfigFileContent.includes(getNpmConfigFileContent(""))) {
-            return;
+            npmConfigContent = npmConfigFileContent.replace(
+                new RegExp(`(@genezio-sdk:registry=https://${GENEZIO_REGISTRY}/npm\n//${GENEZIO_REGISTRY}/:_authToken=)(.*)(\n?)`),
```

Comment:
I would remove `@genezio-sdk:registry=https://${GENEZIO_REGISTRY}/npm\n`

Reply:
Why? How would that work?

Reply:
@vladiulianbogdan The lines do not necessarily come one after another. I have made a commit that treats this case. Now, the lines are deleted independently and the update is only done only on the `//${GENEZIO_REGISTRY}/:_authToken=.*` line.